<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordMainWindow</class>
 <widget class="QMainWindow" name="RecordMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>高清录直播系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget, QLabel{
                        color:white;
                        border: 1px solid white;
                        padding:0;
                        border-radius:5px;
							background-color:rgb(20,50,70);
                      /**  background-color:rgba(250,170,0,150);**/
  }

/*TabWidget*/
QTabWidget{
/*给tabwidget设置etAttribute(Qt::WA_StyledBackground)*/
border: 1px solid white;
background-color:rgb(20,50,70);
}
QTabWidget::tab-bar{
alignment:left;
left:1px;
}
QTabWidget::pane { 
/*background-color:rgb(20,50,70);*/
border:1px solid white;
} 
QTabBar{
/*只有有按钮的地方才是tabbar，空白处是tabwidget的*/
background-color:white;
}
QTabBar::tab{/*页签*/
min-height:28px;
padding:0 10px;
border:0;
margin:1px 1px 0 0;
background-color:rgb(20,50,70);
}
QTabBar::tab:first{
margin-left:1px;
}
QTabBar::tab:hover{
color:cyan;
}
QTabBar::tab:selected{
background-color: rgb(20, 100, 150);
}
QTabBar::tab:selected:hover{
}

QTabBar::tear{
}
QTabBar::scroller{
}
QMenu {
     background-color: rgb(20,50,70); /* sets background of the menu 设置整个菜单区域的背景色，我用的是白色：white*/
     border: 1px solid white;/*整个菜单区域的边框粗细、样式、颜色*/
 }
  
 QMenu::item {
     /* sets background of menu item. set this to something non-transparent
         if you want menu color and menu item color to be different */
     background-color: transparent;
     padding:8px 32px;/*设置菜单项文字上下和左右的内边距，效果就是菜单中的条目左右上下有了间隔*/
     margin:0px 8px;/*设置菜单项的外边距*/
     border-bottom:1px solid #DBDBDB;/*为菜单项之间添加横线间隔*/
 }
  
 QMenu::item:selected { /* when user selects item using mouse or keyboard */
     background-color: #2dabf9;/*这一句是设置菜单项鼠标经过选中的样式*/
 }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="m_b_rec">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>730</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_pause">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>730</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_zb">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>680</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>同步直播</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_res">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>730</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>同步录制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>730</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_hh">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>660</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>660</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>：</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_min">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>660</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>660</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>：</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_sec">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>660</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QFrame" name="vdMain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1081</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_c_vr">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>670</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>虚拟演播</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_c_screen">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>720</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>屏幕共享</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1134</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
